{"uuid":"feef212b-6922-4c5a-b08f-39889fe33572","name":"Command line suite","children":["97822f77-9296-4399-8f5d-4163a89a754d"],"befores":[{"name":"beforeSuite","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Syntax error: Encountered \"s\" at line 1, column 110.","trace":"java.sql.SQLSyntaxErrorException: Syntax error: Encountered \"s\" at line 1, column 110.\n\tat org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)\n\tat org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)\n\tat org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)\n\tat org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)\n\tat org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)\n\tat org.apache.derby.impl.jdbc.ConnectionChild.handleException(Unknown Source)\n\tat org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)\n\tat org.apache.derby.impl.jdbc.EmbedStatement.execute(Unknown Source)\n\tat FrameWork.DataBaseActivity.insertDataIntoTable(DataBaseActivity.java:243)\n\tat FrameWork.DataBaseActivity.dumpDataIntoDatabase(DataBaseActivity.java:171)\n\tat FrameWork.Framework.beforeSuite(Framework.java:716)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)\n\tat org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1218)\n\tat org.testng.TestNG.runSuites(TestNG.java:1133)\n\tat org.testng.TestNG.run(TestNG.java:1104)\n\tat FrameWork.TestRunner.main(TestRunner.java:62)\nCaused by: ERROR 42X01: Syntax error: Encountered \"s\" at line 1, column 110.\n\tat org.apache.derby.iapi.error.StandardException.newException(Unknown Source)\n\tat org.apache.derby.iapi.error.StandardException.newException(Unknown Source)\n\tat org.apache.derby.impl.sql.compile.ParserImpl.parseStatementOrSearchCondition(Unknown Source)\n\tat org.apache.derby.impl.sql.compile.ParserImpl.parseStatement(Unknown Source)\n\tat org.apache.derby.impl.sql.GenericStatement.prepMinion(Unknown Source)\n\tat org.apache.derby.impl.sql.GenericStatement.prepare(Unknown Source)\n\tat org.apache.derby.impl.sql.conn.GenericLanguageConnectionContext.prepareInternalStatement(Unknown Source)\n\t... 22 more\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1709112305696,"stop":1709112309256}],"afters":[{"name":"SessionTearDown","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1709112309512,"stop":1709112309524}],"links":[],"start":1709112305687,"stop":1709112309524}